### ISSUE TITLE
Request: Flatpak/Appimage binaries

### ISSUE THREAD

**Issue by PhoenixBlade24** [lindbergh-loader/lindbergh-loader#17]
As Teknoparrot has no native Linux binaries, and likely never will, I am glad to see a project like this exists!. The Sega Lindbergh was a great platform with a lot of great games made for it!. Unfortunately, as an Arch Linux-based user, I do not see a way to build or run this emulator myself. As a result, I would like to request a Flatpak and/or Appimage binary in the future, one that allows users across all distributions the chance to play these great games without much manual setup. This may be a long-term milestone, but it would be great to see!.

**Comment by Caviar-X** [comment]
There are now some issues about this.
1. The `lindbergh` executable wasn't designed to use as a system-wide executable,it is only useable when it is in the same path as the roms do.
2. We're still unable to let most games even playable in other linux distros.
3. As bobbydilley said before,this project is only for testing purpose and it is not designed to be an alternative to teknoparrot.

**Comment by aderumier** [comment]
Hi, I'll try to work on this next month, looking to add games to batocera linux distro.

I have thinking about to make 1container with both loader+rom   for each  game.

**Comment by bobbydilley** [comment]
@aderumier This is super interesting, please keep me updated! :)

**Comment by accumb1s** [comment]
Hello, thank you so much Bobby, i was looking for this since years,  It launches well under MXLinux i had to make a symbolic link: sudo ln -s libglut.so.3.12.0 libglut.so.3.
I can't change the resolution, it's still in 640X480, sometimes for the hotd4 game the windows is top+left in the corner and i can not get it.
I can not run After Burner Climax (ERROR ABS01)
![abc](https://github.com/user-attachments/assets/ce5b2006-4c17-42ce-8569-604296da5199)
FA, Thanx again !

**Comment by bobbydilley** [comment]
We should hopefully be able to look at this more soon as there is now more reliance on SDL and FAudio.

**Comment by Jitterdoomer** [comment]
Yeah that's what I need because my Steam Deck which ran on Arch has a broken pacman due to incomplete OS update install.

**Comment by sammargh** [comment]
I can confirm with a sane pacman setup, all available lib32 library dependencies, and the appropriate missing lib32 libraries built using makepkg-
https://aur.archlinux.org/packages/lib32-faudio
https://aur.archlinux.org/packages/lib32-libstdc++5

That the loader can mostly function on Steam Deck. The only game I've been able to fully launch is Outrun 2 SP but it is working and playable. There's an issue when /bin/sh is invoked that it cannot load lindbergh.so, because arch only provides sh as a 64-bit binary. I am unsure if this is related to the other games not functioning.

**Comment by bobbydilley** [comment]
> There's an issue when /bin/sh is invoked that it cannot load lindbergh.so, because arch only provides sh as a 64-bit binary.

This is common, and shouldn't be the reason the game isn't loading.

**Comment by dkeruza-neo** [comment]
In Arch, some games will not run like for example HOD4. That is why we recommend using distrobox.

**Comment by dkeruza-neo** [comment]
Added a flatpak bundle to the release.
